common/src/deployment.rs

```diff
@@ -29,14 +29,16 @@ pub enum Environment {
 }
 
 pub const DEPLOYER_END_MSG_STARTUP_ERR: &str = "Service startup encountered an error";
+pub const DEPLOYER_END_MSG_BUILD_ERR: &str = "Is the Shuttle runtime missing?\"} service build encountered an error";
 pub const DEPLOYER_END_MSG_CRASHED: &str = "Service encountered an error and crashed";
 pub const DEPLOYER_END_MSG_STOPPED: &str = "Service was stopped by the user";
 pub const DEPLOYER_END_MSG_COMPLETED: &str = "Service finished running all on its own";
+pub const DEPLOYER_RUNTIME_START_RESPONSE: &str = "{response=\"StartResponse { success: ";
```

Comment:
These should be used both for printing the message and for matching it. I'm updating the StartResponse message in my PR, so I can make these changes.

Reply:
Have this on my PR now

Reply:
No problem, will update its usage from `cargo-shuttle`.

Reply:
NVM, as you pointed, they are already covered in the merged PR. Resolving this.